LOG:  database system was shut down at 2015-01-01 14:01:36 EST
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
FATAL:  database "pyperfin" does not exist
FATAL:  database "gov" does not exist
FATAL:  database "records" does not exist
FATAL:  database "mzakany" does not exist
ERROR:  column "spclocation" does not exist at character 32
STATEMENT:  SELECT oid, spcname, spcowner, spclocation, spcacl, shobj_description(oid, 'pg_tablespace') AS comment FROM pg_tablespace
ERROR:  insert or update on table "account" violates foreign key constraint "fk_account__belongs_to"
DETAIL:  Key (belongs_to)=(1) is not present in table "transactions".
STATEMENT:  insert into "public"."account" ( "id", "name", "belongs_to") values ( '1', 'Fifth Third Bank', '1')
ERROR:  insert or update on table "account" violates foreign key constraint "fk_account__belongs_to"
DETAIL:  Key (belongs_to)=(1) is not present in table "transactions".
STATEMENT:  insert into "public"."account" ( "id", "name", "belongs_to") values ( '1', 'Fifth Third Bank', '1')
ERROR:  cannot drop table transactions because other objects depend on it
DETAIL:  constraint fk_account__belongs_to on table account depends on table transactions
HINT:  Use DROP ... CASCADE to drop the dependent objects too.
STATEMENT:  drop table "public"."transactions"
ERROR:  cannot drop table transactions because other objects depend on it
DETAIL:  constraint fk_account__belongs_to on table account depends on table transactions
HINT:  Use DROP ... CASCADE to drop the dependent objects too.
STATEMENT:  drop table "public"."transactions"
ERROR:  column "account" does not exist
STATEMENT:  CREATE INDEX "idx_transactions__account" ON "transactions" ("account")
ERROR:  database "perfin" is being accessed by other users
DETAIL:  There are 5 other sessions using the database.
STATEMENT:  DROP DATABASE perfin;
	
ERROR:  column "spclocation" does not exist at character 32
STATEMENT:  SELECT oid, spcname, spcowner, spclocation, spcacl, shobj_description(oid, 'pg_tablespace') AS comment FROM pg_tablespace
ERROR:  syntax error at or near "select" at character 22
STATEMENT:  SELECT * from perfin
	select * from postgres
	SELECT * FROM perfin;
ERROR:  relation "perfin" does not exist at character 15
STATEMENT:  select * from perfin;
ERROR:  relation "perfin" does not exist at character 15
STATEMENT:  SELECT * FROM perfin;
ERROR:  relation "accounts" does not exist at character 15
STATEMENT:  SELECT * FROM Accounts;
ERROR:  duplicate key value violates unique constraint "transactions_pkey"
DETAIL:  Key (id)=(1) already exists.
STATEMENT:  INSERT INTO "transactions" ("date", "name", "amount", "account") VALUES ('12/15/2015', 'test2', 1115.87, 2) RETURNING "id"
ERROR:  syntax error at or near "SELECT" at character 22
STATEMENT:  SELECT * FROM perfin
	SELECT * FROM perfin;
postgres cannot access the server configuration file "/Users/mzakany/Desktop/python_perfin/db/perfin/postgresql.conf": No such file or directory
LOG:  database system was interrupted; last known up at 2015-01-03 19:22:22 EST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  record with zero length at 0/1C13640
LOG:  redo is not required
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  column "spclocation" does not exist at character 32
STATEMENT:  SELECT oid, spcname, spcowner, spclocation, spcacl, shobj_description(oid, 'pg_tablespace') AS comment FROM pg_tablespace
ERROR:  operator does not exist: text = integer at character 142
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:  SELECT "t"."id", "t"."date", "t"."name", "t"."amount", "t"."account"
	FROM "transactions" "t", "account" "account-1"
	WHERE "account-1"."name" = 1
	  AND "t"."account" = "account-1"."id"
	ORDER BY "t"."date" DESC
ERROR:  operator does not exist: text = integer at character 142
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:  SELECT "t"."id", "t"."date", "t"."name", "t"."amount", "t"."account"
	FROM "transactions" "t", "account" "account-1"
	WHERE "account-1"."name" = 1
	  AND "t"."account" = "account-1"."id"
	ORDER BY "t"."date" DESC
ERROR:  operator does not exist: text = integer at character 142
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:  SELECT "t"."id", "t"."date", "t"."name", "t"."amount", "t"."account"
	FROM "transactions" "t", "account" "account-1"
	WHERE "account-1"."name" = 1
	  AND "t"."account" = "account-1"."id"
	ORDER BY "t"."date"
ERROR:  operator does not exist: text = integer at character 142
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:  SELECT "t"."id", "t"."date", "t"."name", "t"."amount", "t"."account"
	FROM "transactions" "t", "account" "account-1"
	WHERE "account-1"."name" = 1
	  AND "t"."account" = "account-1"."id"
	ORDER BY "t"."date" DESC
ERROR:  syntax error at or near "select" at character 28
STATEMENT:  select * from transactions
	select * from transactions;
postgres cannot access the server configuration file "/Users/mzakany/Desktop/python_perfin/db/perfin/postgresql.conf": No such file or directory
LOG:  database system was interrupted; last known up at 2015-01-10 11:18:56 EST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  record with zero length at 0/1C8E358
LOG:  redo is not required
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  syntax error at or near "from" at character 1
STATEMENT:  from transactions import *;
ERROR:  column "spclocation" does not exist at character 32
STATEMENT:  SELECT oid, spcname, spcowner, spclocation, spcacl, shobj_description(oid, 'pg_tablespace') AS comment FROM pg_tablespace
FATAL:  role "mzakany23" does not exist
FATAL:  role "mzakany23" does not exist
FATAL:  database "pyperfin" does not exist
ERROR:  column "web" does not exist at character 44
STATEMENT:  select * from Transactions
	where name like WEB;
ERROR:  syntax error at or near "clear" at character 1
STATEMENT:  clear
	select * from Accounts;
ERROR:  relation "accounts" does not exist at character 15
STATEMENT:  select * from Accounts;
ERROR:  syntax error at or near "Transactions" at character 53
STATEMENT:  select * from Account where name = 'Chase' AND JOIN Transactions;
ERROR:  syntax error at or near "JOIN" at character 44
STATEMENT:  select * from Account where name = 'Chase' JOIN Transactions;
ERROR:  syntax error at or near ";" at character 41
STATEMENT:  SELECT * FROM Transactions JOIN Account
	;
ERROR:  syntax error at or near "JOIN" at character 1
STATEMENT:  JOIN * FROM account JOIN transactions;
ERROR:  syntax error at or near "JOIN" at character 1
STATEMENT:  JOIN * FROM Account JOIN Transactions;
ERROR:  syntax error at or near ";" at character 40
STATEMENT:  SELECT * FROM Account JOIN Transactions;
ERROR:  syntax error at or near "JOIN" at character 38
STATEMENT:  SELECT * FROM Account WHERE name = 1 JOIN Transactions;
ERROR:  syntax error at or near ";" at character 40
STATEMENT:  SELECT * FROM Account JOIN Transactions;
ERROR:  syntax error at or near ";" at character 38
STATEMENT:  SELECT * FROM Account JOIN belongs_to;
ERROR:  syntax error at or near ";" at character 43
STATEMENT:  select * from Transactions JOIN belongs_to;
ERROR:  operator does not exist: text >= integer at character 56
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:   SELECT * from Transactions WHERE account = 2 and date BETWEEN 12/01/2014 AND 12/31/2014; 
ERROR:  operator does not exist: text >= integer at character 56
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:   SELECT * from Transactions WHERE account = 2 and date BETWEEN 12/01/2014 AND 12/31/2014; 
ERROR:  operator does not exist: text >= integer at character 56
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:   SELECT * from Transactions WHERE account = 2 and date BETWEEN 12/01/2014 AND 12/31/2014; 
ERROR:  operator does not exist: text >= integer at character 58
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:   SELECT * from Transactions WHERE account = (2) and date BETWEEN (12/01/2014) AND (12/31/2014); 
ERROR:  operator does not exist: text >= integer at character 58
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:   SELECT * from Transactions WHERE account = (2) and date BETWEEN (12/01/2014) AND (12/31/2014); 
ERROR:  operator does not exist: text >= integer at character 58
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
STATEMENT:   SELECT * from Transactions WHERE account = (2) and date BETWEEN (12/01/2014) AND (12/31/2014); 
LOG:  unexpected EOF on client connection with an open transaction
LOG:  received smart shutdown request
LOG:  autovacuum launcher shutting down
LOG:  shutting down
LOG:  database system is shut down
LOG:  database system was shut down at 2015-01-15 14:14:23 EST
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  syntax error at or near "flask_db" at character 6
STATEMENT:  drop flask_db
	;
ERROR:  table "flask_db" does not exist
STATEMENT:  DROP TABLE flask_db
	;
FATAL:  database "flask_db" does not exist
LOG:  received smart shutdown request
LOG:  autovacuum launcher shutting down
LOG:  shutting down
LOG:  database system is shut down
LOG:  database system was shut down at 2015-01-15 14:20:29 EST
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  column "spclocation" does not exist at character 32
STATEMENT:  SELECT oid, spcname, spcowner, spclocation, spcacl, shobj_description(oid, 'pg_tablespace') AS comment FROM pg_tablespace
LOG:  received smart shutdown request
LOG:  autovacuum launcher shutting down
FATAL:  the database system is shutting down
FATAL:  the database system is shutting down
FATAL:  the database system is shutting down
LOG:  shutting down
LOG:  database system is shut down
LOG:  database system was shut down at 2015-01-15 21:14:57 EST
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
FATAL:  database "flask" does not exist
FATAL:  database "flask" does not exist
LOG:  received smart shutdown request
LOG:  autovacuum launcher shutting down
LOG:  shutting down
LOG:  database system is shut down
LOG:  could not bind IPv6 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
LOG:  could not bind IPv4 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
LOG:  could not bind IPv6 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
WARNING:  could not create listen socket for "localhost"
FATAL:  could not create any TCP/IP sockets
LOG:  could not bind IPv6 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
LOG:  could not bind IPv4 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
LOG:  could not bind IPv6 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
WARNING:  could not create listen socket for "localhost"
FATAL:  could not create any TCP/IP sockets
LOG:  database system was shut down at 2015-01-18 10:07:02 EST
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  column "spclocation" does not exist at character 32
STATEMENT:  SELECT oid, spcname, spcowner, spclocation, spcacl, shobj_description(oid, 'pg_tablespace') AS comment FROM pg_tablespace
LOG:  database system was interrupted; last known up at 2015-01-18 12:52:43 EST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  record with zero length at 0/1C94230
LOG:  redo is not required
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
postgres cannot access the server configuration file "/Users/mzakany/Desktop/python_perfin/db/perfin/postgresql.conf": No such file or directory
FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 909) running in data directory "/Users/mzakany/Desktop/python_perfin/db/pyperfin"?
